### ISSUE TITLE
Ability to change show password font/weight

### ISSUE THREAD

**Issue by carlinmack** [PhilippC/keepass2android#1254]
The font currently used when displaying passwords - when unlocking or viewing entries - is the very nice and legible Roboto Mono. However I find this a bit too legible, I can read this text from ~1.5m away. I would prefer if this was an ultra-light font or the thin courier that is used in the desktop KeePass app

I've tried to recreate what the font in the app looks like compared to the desktop app.
![image](https://user-images.githubusercontent.com/6676843/82152486-81653580-982f-11ea-8e0d-15fc79cd7233.png)
Fonts are 8pt Courier and 12.5pt Roboto Mono

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Monospaced ist important. Also let see difference between 0O lI homoglyphs. 
I like the idea how the desktop keepass plugin password syntax highlighting solved it.

**Comment by JohnAZoidberg** [comment]
> Also let see difference between 0O lI homoglyphs.

Yes, I can never tell 0 from O in the password field. Sometimes I use the app to look at my password and type it into a different device. That's very difficult if I can't read the password.

**Comment by PhilippC** [comment]
@JohnAZoidberg Zeros have a dot inside, capital O does not. If you can't remember which is which, think of "zero dot zero".
